### DISCUSSION TITLE
Anzeige bei 3 Phasen inverter (HMT-2250)

### DISCUSSION THREAD

**Discussion by bf4ed** [tbnobody/OpenDTU#1303] [General]
Ich habe einen HMT-2250 (3 Phasen) und bei der anzeige auf der ersten seite hab ich nur Phase1.

Wo sehe ich die anderen 2 Phasen?

![Screenshot_20230902-180755_cropped](https://github.com/tbnobody/OpenDTU/assets/130218342/5dbd35e9-f2ab-43a3-a3e0-eb351d33b10c)

**Comment by jlf-cleaner** [comment]
Bei mir dasselbe, habe 2x HMS 2250 eingerichtet, bei beiden wird nur Phase 1 gezeigt.

**Comment by desiue** [comment]
Hallo zusammen,

ich habe das gleiche Problem. 
Wechselrichter HMT-2250-6T und OpenDTU/CMT2300A  mit Firmwareversion / git Hash | v24.2.16 
(Habe gesehen das es seit 12 Stunden eine neue Firmware gibt v24.3.22, habe geupdatet, keine Besserung)
Es wird nur eine Phase Angezeigt gibt's eine Lösung?

Da das Haus leer steht ist die StandBy Last bei 65Watt. 
Dabei ist mir aufgefallen das die Summe der Watt Zahlen nie stimmt.
z.B.
73 Watt musst die DTU als Leistung anzeigen damit am Stromzähler die 65 Watt auf 0 kommen. 
Das sind 12% finde ich als Abweichung etwas viel.
Den Stromzähler MT631 lese ich mit SML/Hichi/Tasmota/ESP32 aus.
Kann man die Genauigkeit anpassen?

Da ich den Selbstverbrauch aus "Erzeugt" (openDTU) abzüglich "Eingespeist" (Stromzähler) berechne sind die Werte entsprechen auch ungenau.
![opendtu2](https://github.com/tbnobody/OpenDTU/assets/77403450/e7410b07-0c4c-4334-adf9-d83898e1880f)



Hier noch ein Live Bild Beispiel. Links OpenDtu rechts tasmota. Grundlast ist 65Watt
80W werden erzeugt - 65W Grundlast = 15W Überschuss angezeigt werden 6 Watt

![opendtu](https://github.com/tbnobody/OpenDTU/assets/77403450/e23b9c6d-0c00-4066-99ca-e055ab08ead4)

**Comment by Charly-2** [comment]
Hallo! 
Ich denke die Kennzahlen unter "Phase 1" beziehen sich auf den gesamten AC-Zweig, also alle 3 Phasen bei HMT und eben nur eine Phase bei HMS.
Hier wäre wohl eine andere Bezeichnung wie z.B: "AC" oder ähnliches weniger verwirrend.
Allerdings bleibt die Frage worauf genau sich 
Spannung (L1-L2?, L2-L3?,L3-L1? - gegen N kann es nicht sein, weil bei meinem HMT 400V angezeigt werden) und 
Strom (L1?, L2?, L3?, N?) genau beziehen?

**Comment by tbnobody** [comment]
> Ich denke die Kennzahlen unter "Phase 1" beziehen sich auf den gesamten AC-Zweig, also alle 3 Phasen bei HMT und eben nur eine Phase bei HMS.

Das ist korrekt. Bis ich die MQTT Struktur auf mehrere Phasen aufgeteilt habe, zeige ich bei Phase 1 den Gesamtstrom an.

> Allerdings bleibt die Frage worauf genau sich Spannung

Im Falle eines HMT (6 Kanal) ist das die Spannung L1-L2. Selbe Begründung wie oben.

**Reply by Kroki0815** [reply]
@tbnobody 
Moin Moin,
ich hab mir auch einen 2250 zugelegt. Hast du schon einen Plan wann die 3-Phasige Anzeige implementiert sein könnte. Wäre echt cool wenn man die Daten aller 3 Phasen auswerten könnte.

Gruß Kroki
